src/elements/review_by_id.jsx

```diff
@@ -22,14 +37,31 @@ function ReviewById() {
     return <h2>loading...</h2>;
   } else {
     return (
```

Comment:
As this if statement is returning in the body, you could omit the else and just say
```suggestion
    return <h2>loading...</h2>;
  } 
    return (
```